Gemfile

```diff
@@ -43,6 +44,10 @@ group :development do
   gem 'rack-mini-profiler'                  # Inline app profiler. See ?pp=help for options.
   gem 'flamegraph', platforms: :mri         # Flamegraph visualiztion: ?pp=flamegraph
 
+  # Flamegraph dependency
+  gem 'stackprof', platforms: [:mri_21, :mri_22, :mri_23], require: false
+  gem 'fast_track', platforms: [:mri_19, :mri_20], require: false
+
```

Comment:
Due: https://github.com/SamSaffron/flamegraph/issues/19
It fails in dummy(it is when i tried with sprockets 4).